sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -540,6 +565,8 @@ private Read() {}
         this.schema = schema;
         this.validate = validate;
         this.codec = codec;
+        this.windowedWrites = windowedWrites;
+        this.filenamePolicy = filenamePolicy;
```

Comment:
Right now they get translated into DefaultFilenamePolicy. We could refactor further and force the user to specify the policy themselves, however I think that's best left to a subsequent PR